scripts/inc_woq_g2_bkc.md

```diff
@@ -38,8 +38,25 @@ This script 1) converts official model weights from `torch.float8_e4m3fn` format
 > 
 >For more details, refer to the `INC WOQ ReQuant` section in the `single_16k_len_inc.sh` script.
 
+
+
+### Configure the Measurement Statistics Results
+
+The environment variable `INC_MEASUREMENT_DUMP_PATH_PREFIX` specifies the root directory where measurement statistics were saved.
+The final path is constructed by joining this root directory with the `dump_stats_path` defined in the quantization JSON file specified by the `QUANT_CONFIG` environment variable.
+
+Example:
+```bash
+INC_MEASUREMENT_DUMP_PATH_PREFIX=/mnt/disk3/vllm-fork
+dump_stats_path (from config): "scripts/nc_workspace_measure_kvache/inc_measure_output"
+Resulting full path: "/mnt/disk3/vllm-fork/scripts/nc_workspace_measure_kvache/inc_measure_output_xx"
```

Comment:
what's the meaning of last _xx? can you add some explanation to make it clearer?

Reply:
Updated, please check again, thx!